test.js

```diff
@@ -0,0 +1,323 @@
+'use strict'
+
+var t = require('tap')
+const stringifyWrapper = require('./index')
```

Comment:
it's not a "wrapper", but rather a "factory"

Reply:
done